code/WorkInProgress/cryospawn.dm

```diff
@@ -168,6 +170,11 @@
 			if (H.sims)
 				for (var/name in H.sims.motives)
 					H.sims.affectMotive(name, 100)
+
+		var/datum/db_record/crew_record = data_core.general.find_record("name", L.real_name)
+		if (!(crew_record == null))
```

Comment:
```suggestion
		if (!isnull(crew_record))
```
Doesn't matter in this case but it's good practice to use `isnull` as for example `0 == null` but `isnull(0)` is not true.